### DISCUSSION TITLE
AttributeError

### DISCUSSION THREAD

**Discussion by Wahomethegeek** [ultrafunkamsterdam/undetected-chromedriver#1590] [Q&A] [answered]
File "/home/wahome/pyinstxtractor/pycdc/build/ZWD.py", line 63, in <module>
    driver = MyUDC()
             ^^^^^^^
  File "/home/wahome/.local/lib/python3.11/site-packages/undetected_chromedriver/__init__.py", line 398, in __init__
    if headless or options.headless:
                   ^^^^^^^^^^^^^^^^
AttributeError: 'ChromeOptions' object has no attribute 'headless'
am getting this error when using python3.11 but python3.10 is working 
what might be the issue

**Comment by Wahomethegeek** [comment]
I am not using headless browser

**Reply by ThomasTrovon** [reply, marked as answer]
The error occurs for everyone who has the latest package updates, regardless of whether they are using Headless or not. 

My Solution is below.

**Comment by ThomasTrovon** [comment]
Here I temporarily solved it by doing this. In this line:

![image](https://github.com/ultrafunkamsterdam/undetected-chromedriver/assets/10778234/14924d28-a123-4007-9565-97d3278038d8)

And after in this line:

![image](https://github.com/ultrafunkamsterdam/undetected-chromedriver/assets/10778234/53a16c6d-a325-4a64-91ce-0fc367fec231)

So, then it worked...

**Comment by pkhubpp8** [comment]
it might help
```
import undetected_chromedriver as uc
from selenium.webdriver.chromium.options import ChromiumOptions
options = ChromiumOptions()
options.headless = False
uc.Chrome(options=options)
```
